client/src/component/ticketForm.test.js

```diff
@@ -1,19 +1,18 @@
-import { render, screen, fireEvent} from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import TicketForm from './ticketForm';
 
-
-test('renders TicketForm', () => {
+describe('ticketForm test', () => {
   render(<TicketForm />);
-  const firstNameInput = screen.getByLabelText('First Name')
-  expect(firstNameInput).toBeInTheDocument();
+  const ticketBtn = screen.getByText('Create Ticket')
 
-});
+  test('renders TicketForm', () => {
+    expect(ticketBtn).toBeInTheDocument();
+  });
 
-test('callApi is called when button is clicked', () => {
+  test('callApi is called when button is clicked', () => {
     const callApi = jest.fn();
-    render(<TicketForm/>)
-    const button = screen.getByText('Create Ticket');
-    button.onclick = callApi;
-     fireEvent.click(button);
-     expect(callApi).toHaveBeenCalled(1);
-})
+    ticketBtn.onclick = callApi;
+    fireEvent.click(ticketBtn);
+    expect(callApi).toHaveBeenCalledTimes(1);
+  });
```

Comment:
I think we left this a little unclear, but if the function isn't passed in as a prop, this test doesn't have much value.

Reply:
Okay, would it be best to get rid of this test case then, or should the code be refactored in the ticketForm.js so that the function is passed in as a prop?

Reply:
I think we can safely get rid of it.